test/CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ foreach(ONNX_TEST ${ONNX_TESTS})
     set(TEST_NAME test_${BASE_NAME})
     add_executable(${TEST_NAME} ${ONNX_TEST})
     rocm_clang_tidy_check(${TEST_NAME})
-    target_link_libraries(${TEST_NAME} migraphx_onnx migraphx_ref)
+    target_link_libraries(${TEST_NAME} migraphx_onnx -Wl,--no-as-needed migraphx_ref)
```

Comment:
This should be `-Wl,--no-as-needed migraphx_ref -Wl,--as-needed`.

Reply:
Done.